app/models/invenio_rdm_record_converter.rb

```diff
@@ -263,25 +263,16 @@ def build_creator_contributor_json(creator)
     json
   end # build_creator_contributor_json
 
-  def additional(category:, array:)
-    type = set_type(category)
-    # return all values except the first
-    tail_values = array.slice(1..-1)
-    return [] if tail_values.blank?
-
-    # remove the strings that contain only spaces or are empty, then map
-    tail_values.delete_if(&:blank?).map do |word|
-      {"#{category}": word, "type": {"id": type, "title": {"en": type.titleize}}}
+  def format_additional(content_type, invenio_type, values)
```

Comment:
`content_type` is probably not the best name, but I figured that "description" or "title" sort of fit it. Passing the values directly was to make it fit existing tests more easily. Rather than calculating tail values here, I moved that out so that whatever calls the function would send the values it wants formatted. Due to this acknowledgements, which does not drop a value, can be mapped with this function as well as title and descriptions which do drop a value.